source/tutorials/test-strategy/test-strategy.rst

```diff
@@ -0,0 +1,77 @@
+.. _system-validation-test-strategy:
+
+System Validation - Test strategy for |C|
+#########################################
+
+|C| is an open source Android software reference stack for the Android community to bring ideas to life while developing on the latest Android release and the latest Intel hardware platforms. The following |NUC| systems are used for validating regular |C| releases.
+
+    * |NUC| Kit `NUC7i5DNHE <https://www.intel.com/content/www/us/en/products/boards-kits/nuc/kits/nuc7i5dnhe.html>`_
+    * |NUC| Kit `NUC6CAYH <https://www.intel.com/content/www/us/en/products/boards-kits/nuc/kits/nuc6cayh.html>`_
+
+Weekly Validation Phase
+-----------------------
+
+Weekly validation cycle includes system functional test execution, Compliance testing - `CTS <https://source.android.com/compatibility/cts>`_/`VTS <https://source.android.com/compatibility/vts>`_/`CTS-Verifier <https://source.android.com/compatibility/cts/verifier>`_/STS and Power and performance KPI's based on the defined features on the program quarterly.
+
+    * The testplan is created based out of the features in verify/closed state on the weekly release builds. The execution is done weekly on each domain until the SWQRC criteria, which is defined at the program start, is met.
+    * HP ALM (Application Lifecycle Management) tool is used to map/track all the testcases against the requirements defined in Jira.
+    * PLC - Product lifecycle portal gives the consolidated results and helps tracking the software completion, requirement coverage, defects and system validation results.
```

Comment:
Line 18 suggested rewrite: 
The PLC (Product Lifecycle Portal) gives the consolidated results in addition to tracking the software completion, requirement coverage, defects, and system validation results.

Reply:
done